draft-irtf-cfrg-frost.md

```diff
@@ -568,10 +568,10 @@ each Signer then runs the following procedure to produce its own signature share
     for (_, hiding_nonce_commitment, binding_nonce_commitment) in commitment_list:
       R = R + (hiding_nonce_commitment + (binding_nonce_commitment * binding_factor))
 
+    # Interpret index and participant_list's elements as x-coordinate scalars in GF(p)
```

Comment:
This is noted in the inputs of `derive_lagrange_coefficients()`

Reply:
Yeah, I'd prefer we drop this comment.

Reply:
OK, I will drop this comment